UKHomeOffice/ide-controller
# Update image display style

## Description
The branch updates the display styles of the chip, scan and live photo image. Setting the display height, adding a shadow to the canvas and adding a white boarder to the image. It also reduces the padding between each image so as to increase display size. 

## How to test
- Install app and run dev (this will open an electron app window)
- If this page is blank, refresh the page with `cmd + r` (mac) or `ctl + r` (windows)
- 2 blank avatar images and one live webcam image will display in the "Images to compare" tab component. These should possess the properties described above. 
- Live webcam image does not yet fit the canvas correctly. 

## Developer Checklist
\* Required
